cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -43,6 +43,13 @@ pub struct CallInfo {
     pub opcode: Opcode,
 }
 
+#[test]
+fn inst_size_test() {
+    // This test will help with unintentionally growing the size
+    // of the Inst enum.
+    assert_eq!(72, std::mem::size_of::<Inst>());
```

Comment:
Could this test have a `#[cfg(target_pointer_width = "64")]` on it just in case we compile for a 32-bit platform one day?

Reply:
Great point, I'll add that!